### PR TITLE
docs: add new organizations docs to best practices section

### PR DESCRIPTION
**Pull request by EdwardAngert**

### CODE REVIEW

**Review by Emyrk — COMMENTED** [review]
Identity mapping to be moved into orgs

organizations-architecture (1)

File: docs/tutorials/best-practices/organizations.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+# Organizations - best practices
+
+December 9, 2024
+
+---
+
+Coder [Organizations](../../admin/users/organizations.md) allow administrators
+finer control over groups, templates, and provisioners within their Coder
```
Comment (Emyrk) [inline docs/tutorials/best-practices/organizations.md:8]: workspaces too?

```suggestion
finer control over groups, templates, workspaces, and provisioners within their Coder
```

**Review by Emyrk — APPROVED** [review]

**Comment by Emyrk** [comment]
Some suggestions, overall approve :+1:

**Comment by EdwardAngert** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/tutorials/best-practices/organizations.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+and groups.
+
+![Organizations architecture](../../images/best-practice/organizations-architecture.png)
+
+Users can belong to multiple organizations while templates and provisioners
+cannot.
+
+Organizations allow multiple platform teams to offer templates and
+infrastructure to their users instead of having them entirely managed in a
+centralized fashion.
```
[the remark this answers is not in the document]
Comment (EdwardAngert) [reply docs/tutorials/best-practices/organizations.md:35]: good catch - I'll remove this one and keep the intro one

**Pull request merged**